core/claims/claimBasic.go

```diff
@@ -1,85 +1,70 @@
 package claims
 
 import (
-	"encoding/binary"
-
 	"github.com/iden3/go-iden3-core/merkletree"
 )
 
 const (
-	// 56+248+248+248=800 bits
-	indexSlotBits  = 800
+	// 152+248+248+248=896 bits
+	indexSlotBits  = 896
```

Comment:
this shouldn't be 120+248+248+248= 864?

Reply:
I used 152 because the version field is not being used in this claim.

Reply:
Right, perfect